src/components/SearchableDropdown.jsx

```diff
@@ -1,6 +1,6 @@
-import React, { useState, useRef, useEffect } from 'react';
+import { useState, useRef, useEffect } from 'react';
 
-const SearchableDropdown = ({ options = [], value, onChange, placeholder }) => {
+export const SearchableDropdown = ({ options = [], value, onChange, placeholder }) => {
```

Comment:
I switched this from a default export to a named export as the general consensus in the community seems to avoid default exports.

https://www.olivare.net/blog/2023/death-to-default-exports